pkg/cloudprovider/provider/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+  - sig-osm
+
+reviewers:
+  - sig-osm
+
+labels:
+  - sig/cluster-management
```

Comment:
You can specify your own label, preferably after adding it to https://github.com/kubermatic/infra/blob/master/prow/label_sync/labels.yaml and then later you can filter github PRs by your own label.

Reply:
Definitely, please use sig/osm

Reply:
Makes sense, done.